providers.js

```diff
@@ -1,15 +1,12 @@
 require("dotenv").config();
 const HDWalletProvider = require("@truffle/hdwallet-provider");
 
-const ganacheMnemonic = process.env["GANACHE_MNEMONIC"];
-const kovanMnemonic = process.env["KOVAN_MNEMONIC"];
-const mnemonic = process.env["MNEMONIC"];
-const mainnetMnemonic = process.env["MAINNET_MNEMONIC"];
```

Comment:
These were duplicate and unused from those listed in the `truffle-config.ovm.js` file